packages/schematics/angular/workspace/files/tslint.json

```diff
@@ -4,22 +4,16 @@
     "codelyzer"
   ],
   "rules": {
-    "adjacent-overload-signatures": false,
-    "align": false,
     "array-type": false,
     "arrow-parens": false,
-    "ban-types": false,
     "deprecation": {
       "severity": "warn"
     },
-    "function-constructor": false,
     "import-blacklist": [
       true,
       "rxjs/Rx"
     ],
     "interface-name": false,
-    "jsdoc-format": false,
-    "max-classes-per-file": false,
```

Comment:
This should probably left as is since it is highly opinionated especially considering the recommended value is one class per file.